html-syntax/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <img src = "https://static.wikia.nocookie.net/spellcasters/images/e/e3/145Zapdos.png/revision/latest?cb=20210101111145">
+  <h1>
+    Zapdos
+  </h1>
```

Comment:
These should be in the `body` element, they are visual content. Additionally, you are missing your `head` element, please include that as well, even if it is empty

Reply:
Thanks @codingcodymiller I have made the changes. Please let me know if theres anything else I need to modify.